Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Action/EditorAction.h

```diff
@@ -43,7 +43,13 @@ namespace AzToolsFramework
         //! Returns the pointer to the action.
         QAction* GetAction();
 
-        //! Calls the callback to update the action's checked state, if any.
+        //! Sets the enabled state callback for the action.
+        void SetEnabledStateCallback(AZStd::function<bool()> enabledStateCallback);
+
+        //! Returns true if the EditorAction has an enabled state callback set, false otherwise.
+        bool HasEnabledStateCallback();
```

Comment:
update: This should probably be `const` (also should we also have an `IsEnabled` call too? Or is that handled elsewhere?)

Reply:
const added, IsEnabled added, changed the ActionManager IsActionEnabled to use IsEnabled.